tests/test_imputation/test_categorical_imputer.py

```diff
@@ -292,3 +292,8 @@ def test_variables_cast_as_category_frequent(df_na):
     assert X_transformed[["City", "Studies"]].isnull().sum().sum() == 0
     assert X_transformed[["Age", "Marks"]].isnull().sum().sum() > 0
     pd.testing.assert_frame_equal(X_transformed, X_reference)
+
+
+def test_error_when_ignore_format_is_not_boolean():
```

Comment:
we'd use parametrize to test more than 1 value here, for example 22.3, "hola", {some dictionary} and we'd also assert that the error message is the expected one.

Reply:
done

Reply:
pls reformat with parametrize